cmd/frontend/graphqlbackend/authz.go

```diff
@@ -159,3 +160,13 @@ const (
 )
 
 type CancelPermissionsSyncJobResultMessage string
+
+type PermissionsSyncingStatsResolver interface {
+	QueueSize(ctx context.Context) (int32, error)
+	UsersWithLatestJobFailing(ctx context.Context) (int32, error)
+	ReposWithLatestJobFailing(ctx context.Context) (int32, error)
+	UsersWithNoPermissions(ctx context.Context) (int32, error)
+	ReposWithNoPermissions(ctx context.Context) (int32, error)
+	UsersWithOldPermissions(ctx context.Context) (int32, error)
+	ReposWithOldPermissions(ctx context.Context) (int32, error)
```

Comment:
I think maybe we can use the word "stale" in this case?
```suggestion
	UsersWithStalePermissions(ctx context.Context) (int32, error)
	ReposWithStalePermissions(ctx context.Context) (int32, error)
```

Reply:
fixed